src/mapper/components/HeaderMapper.tsx

```diff
@@ -70,18 +70,18 @@ export default function HeaderMapper({
         );
       })}
       <div className="my-5">
-        <Row>
-          <Col>
+        <div className="flex">
```

Comment:
<img width="532" alt="Screenshot 2025-02-14 at 8 08 29 AM" src="https://github.com/user-attachments/assets/ef213533-c97e-4cca-9c01-22849e6ff78b" />


Won't that achieve the same effect with less code?